crates/ruma-common/src/time.rs

```diff
@@ -28,6 +30,16 @@ impl MilliSecondsSinceUnixEpoch {
     pub fn to_system_time(self) -> Option<SystemTime> {
         UNIX_EPOCH.checked_add(Duration::from_millis(self.0.into()))
     }
+
+    /// Get the unix epoch in milliseconds
```

Comment:
Other than this small nit LGTM!

```suggestion
    /// Get the unix epoch in milliseconds.
```

Reply:
i guess also for the other docs :)

Reply:
Yeah good catch I meant to mention all of them :relieved: